app/screens/DashboardEditScreen.tsx

```diff
@@ -8,31 +8,41 @@ import { BackHeader } from "@/components/BackHeader"
 import { BodyText } from "@/components/BodyText"
 import { DashboardSettingsCard } from "@/components/DashboardSettingsCard"
 import { Screen } from "@/components/Screen"
-import { Text } from "@/components/Text"
 import { AppStackScreenProps } from "@/navigators/AppNavigator"
 import { useDashboardPreferences } from "@/services/hooks/useDashboardPreferences"
+import { usePostDashboardPreferences } from "@/services/hooks/useDashboardPreferences"
 import { tamaguiConfig } from "@/tamagui.config"
 import { spacing } from "@/theme/spacing"
 import { $styles } from "@/theme/styles"
 
+import { DashboardPreferences } from "../../backend/src/types/settings.types"
+
 const initialCards = [
-  { id: "duty", title: "Who's on duty", subtitle: "See who's on duty based on your saved filters" },
   {
-    id: "upcomingShift",
+    id: "whos_on_duty",
+    title: "Who's on duty",
+    subtitle: "See who's on duty based on your saved filters",
+  },
```

Comment:
I think we can type this a bit better, we know that the ID must be a key of `DashboardPreferences`, not just any string, and this should help type inference later on

```suggestion
type DashboardCard = {
  id: keyof DashboardPreferences
  title: string
  subtitle: string
}

const DASHBOARD_CARDS: DashboardCard[] = [
  {
    id: "whos_on_duty",
    title: "Who's on duty",
    subtitle: "See who's on duty based on your saved filters",
  }, // etc.
```